spot-webdriver/specs/adhoc-meeting.spec.js

```diff
@@ -14,7 +14,7 @@ describe('Can start a meeting', () => {
         expect(spotTV.getMeetingName()).toBe(testMeetingName);
     });
 
-    it('and disconnects the remote on meeting end', () => {
+    xit('and disconnects the remote on meeting end', () => {
```

Comment:
what is the purpose of that? it says "disable while feature is disabled", but it seems to be disabling it permanently, no? How is the feature being enabled figured out?

Reply:
It's enabled through the config. I could have the test read the config and figure it out from there, if that's what you're implying.

Reply:
was your plan to enable it later or what? I just dont see a way how this test can be used

Reply:
Yeah my plan was to un-comment out the test after the feature flag is removed.

Reply:
this test flag will be false on PR tests anyway, right? so I think it's fine this way

Reply:
Yes, the test flag will be false and the test itself is testing the new feature specifically. I could mock out the selector for the flag. Let me look into that.